app/javascript/controllers/subscribe_links_controller.js

```diff
@@ -0,0 +1,33 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static outlets = ["unsaved"]
+  static targets = ["template"]
+
+  addLink(event) {
+    const now = new Date().getTime()
+    const platform = event.target.dataset.platform
+
+    const template = this.templateTargets.filter((target) => {
+      return target.dataset.platform === platform
+    })
+    const html = template[0].innerHTML.replaceAll("NEW_RECORD", now).replaceAll("NEW_EXTERNAL_ID", "")
+    this.templateTarget.insertAdjacentHTML("beforeBegin", html)
+    this.unsavedOutlet.change()
+    event.target.classList.add("d-none")
```

Comment:
largely follows other such patterns like `FeedTokenController`, with this last addition to hide the option once it's selected.